client/src/services/dialog/dialog.component.html

```diff
@@ -0,0 +1,19 @@
+<div>
+  <form class="modal-content container">
+    <span class="close" [mat-dialog-close]="false">&times;</span>
+    <mat-dialog-content class="mat-typography">
+      <h2 mat-dialog-title class="dialog-title">{{ data.title }}</h2>
+      <p class="dialog-message">{{ data.message }}</p>
+      <b> Deletion is irreversible and may cause unwanted behavior</b>
+    </mat-dialog-content>
+    <hr />
+    <mat-dialog-actions class="clearfix">
+      <button mat-button class="cancelbtn" [mat-dialog-close]="false">
```

Comment:
Script this according to the `type` of the dialog. For example, only show the cancel button if the dialog is of type `Confirmation`.